test/custom_compose_preset.yml

```diff
@@ -0,0 +1,16 @@
+databases:
+    - compose:
+          cpu_count: 2
+          mac_address: 02:42:ac:11:65:43
+          deploy:
+              resources:
+                  reservations:
+                      memory: 4G
+nodes:
```

Comment:
Just curious.  What does reservation do?

Reply:
I was expecting to start memory from this value (min) but it doesn't. Limit does limit the max value.

So, we are not 100% sure the best values so I would live it empty for now. I'll update